response_operations_ui/views/messages.py

```diff
@@ -77,20 +77,14 @@ def view_conversation(thread_id):
     thread_conversation = message_controllers.get_conversation(thread_id)
     refined_thread = [_refine(message) for message in reversed(thread_conversation['messages'])]
     latest_message = refined_thread[-1]
+    closed_at = _format_closed_at(thread_conversation)
+    breadcrumbs = _get_conversation_breadcrumbs(thread_conversation['messages'])
+    respondent_is_deleted = False
 
-    try:
-        closed_time = localise_datetime(datetime.strptime(thread_conversation['closed_at'], "%Y-%m-%dT%H:%M:%S.%f"))
-        closed_at = closed_time.strftime("%d/%m/%Y" + " at %H:%M")
-    except KeyError:
-        closed_at = None
-
-    try:
-        breadcrumbs = _get_conversation_breadcrumbs(thread_conversation['messages'])
-    except IndexError:
-        breadcrumbs = [
-            {"text": "Messages", "url": "/messages"},
-            {"text": "Unavailable"}
-        ]
+    for message in refined_thread:
+        # This needs to change once we figure out what the name of a missing/deleted user is going to be.
+        if 'None - ' in message['username']:
```

Comment:
Will this prevent us from replying to message threads if a different internal ONS user involved in the message thread has their account deleted? That's obviously not functionality we have at the moment but something to consider going forward

Reply:
Currently only respondents get this message happening.  If an ONS user is deleted/not there, then the call fails in a different, more obvious way